### PR TITLE
Move ServerBrowseUriEvent out of MD events

### PR DESCRIPTION
**Pull request by ARTI1208**
Client changes: https://github.com/JetBrains/projector-client/pull/97

### CODE REVIEW

File: projector-server/src/main/kotlin/org/jetbrains/projector/server/ProjectorServer.kt [hunk 1]
```diff
@@ -116,6 +116,8 @@ class ProjectorServer private constructor(
 
   private val markdownQueue = ConcurrentLinkedQueue<ServerMarkdownEvent>()
 
+  private val baseQueue = ConcurrentLinkedQueue<ServerEvent>()
```
Comment (SerVB) [inline projector-server/src/main/kotlin/org/jetbrains/projector/server/ProjectorServer.kt:119]: I'm not sure, there are too many queues now, but this one is just generic. Maybe it's better to make it specific as other queues:

```suggestion
  private val browseUriQueue = ConcurrentLinkedQueue<ServerBrowseUriEvent>()
```

**Review by SerVB — APPROVED** [review]
Not much changes here, just a small refactoring.

But I can leave a comment.

File: projector-server/src/main/kotlin/org/jetbrains/projector/server/ProjectorServer.kt (hunk 1 above)
Comment (ARTI1208) [reply projector-server/src/main/kotlin/org/jetbrains/projector/server/ProjectorServer.kt:119]: I don't see a big difference between two. Maybe all queues should be just merged in one

Comment (SerVB) [reply projector-server/src/main/kotlin/org/jetbrains/projector/server/ProjectorServer.kt:119]: I don't mind merging the queues. However `baseQueue` still sounds unclear. Maybe we should name that merged queue like `highLevelEvents` (there are caret, markdown, and uri events) or just `commonQueue` (this queue contains the events that should go to all the connected users)

Comment (ARTI1208) [reply projector-server/src/main/kotlin/org/jetbrains/projector/server/ProjectorServer.kt:119]: I like `commonQueue`. Will update the PR at the end of the week

**Pull request merged**